### PR TITLE
Update quickstart.md

### PR DESCRIPTION
**Pull request by lwr20**
Make it clear that one of the instructions is only valid for operator installs.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: windows-calico/quickstart.md [hunk 1]
```diff
@@ -103,11 +103,13 @@ The following steps install a Kubernetes cluster on a single Windows node, with
   <%
 
 1. Ensure that BGP is disabled.
+   If you installed using tigera-operator, you can do this by:
```
Comment (song-jiang) [inline windows-calico/quickstart.md:106]: `If you installed Calico using operator`?

File: windows-calico/quickstart.md [hunk 2]
```diff
@@ -103,11 +103,13 @@ The following steps install a Kubernetes cluster on a single Windows node, with
...
 
    ```bash
    kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
    ```
-  
+   If you installed using the manifest from https://docs.projectcalico.org/manifests/calico-vxlan.yaml then BGP will already be disabled.
```
Comment (song-jiang) [inline windows-calico/quickstart.md:111]: How about `BGP is already disabled`?

Comment (song-jiang) [reply windows-calico/quickstart.md:111]: `If you installed Calico`

**Review by song-jiang — CHANGES_REQUESTED** [review]
Two suggestions.

File: windows-calico/quickstart.md (hunk 1 above)
Comment (lwr20) [reply windows-calico/quickstart.md:106]:
```suggestion
   If you installed Calico using operator, you can do this by:
```

File: windows-calico/quickstart.md (hunk 2 above)
Comment (lwr20) [reply windows-calico/quickstart.md:111]:
```suggestion
   If you installed Calico using the manifest from https://docs.projectcalico.org/manifests/calico-vxlan.yaml then BGP is already disabled.
```

**Review by song-jiang — APPROVED** [review]
LGTM

**Pull request merged**